README.md

```diff
@@ -13,6 +13,19 @@ npm install
 npm start
 ```
 
+## Tasks
+
+- `npm start` Run web server.
+- `npm test` Run automated tests.
+- `npm build` Build project.
```

Comment:
This should be `npm run build` and I'm wondering if we should change the task to run both `less` and `js` because right now it only compile `js` here.